packages/office-addin-cli/src/npmPackage.ts

```diff
@@ -0,0 +1,36 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license.
+
+import readPackageJson, { ScriptsObject } from "read-package-json-fast";
+
+let scripts: ScriptsObject | undefined;
```

Comment:
Try not to use "any". You can use "unknown", but it is better to be able to describe type info more accurately. Here I was able to define ScriptsObject as the type, which is an object whose keys and values are strings.

Reply:
Nice! I found a solution in the internet that was using any, but this ScriptsObject is way better!